apps/omg/test/omg/dependency_conformance/signature_test.exs

```diff
@@ -20,6 +20,7 @@ defmodule OMG.DependencyConformance.SignatureTest do
 
   alias OMG.DevCrypto
   alias OMG.Eth
+  alias OMG.Eth.Test.Support.DevNode
```

Comment:
I did a quick scan of prior art like phoenix/ecto etc, that use the `test/support` dir structure like us.

I think including `Test` is quite nonstandard, `Support` gets included sometimes. If you want a qualifier here, I'd opt for just `Support`. If not that much then just let's drop both and try to have the actual module name actually used around (`Eth.DevNode`) meaningful?

Reply:
@pdobacz resolved this by going with moving this to a top level `Support` module.